test/app/presenter/ChessBoardPresenter.spec.ts

```diff
@@ -6,18 +6,27 @@ import { ViewEventBus } from '../../../src/app/view/events/ViewEventBus';
 import { ViewEvent } from '../../../src/app/view/events/ViewEvent';
 import { SquareWasClicked } from '../../../src/app/view/events/SquareWasClicked';
 import { InMemoryViewEventBus } from '../../../src/app/view/events/InMemoryViewEventBus';
+import { PiecesBoardPositions } from '../../../src/app/model/Types';
+import { PIECES_START_POSITION } from '../../../src/app/model/Constances';
 
 describe('ChessBoardPresenter', () => {
   const viewEvents: ViewEventBus = new InMemoryViewEventBus();
   const view: ChessBoardView = chessBoardViewMock(viewEvents);
-  const model: ChessModel = { move: jest.fn() };
+  const piecesPositions: PiecesBoardPositions = PIECES_START_POSITION;
+  const model: ChessModel = { piecesPositions, move: jest.fn() };
   const presenter: ChessBoardPresenter = new ChessBoardPresenter(view, model);
 
   it('when square was clicked on the view, then selected piece should be shown', () => {
     viewEvents.publish(new SquareWasClicked({ x: 1, y: 1 }));
 
     expect(view.showSelectedPiece).toHaveBeenCalledWith({ x: 1, y: 1 });
   });
+
+  it('when game starts, check if pieces will show on the screen', () => {
+    presenter.startGame();
+
+    expect(view.showChessBoard).toHaveBeenCalledWith(piecesPositions);
```

Comment:
Może tutaj nawet warto zrobić tak:
```suggestion
    expect(view.showChessBoard).toHaveBeenCalledWith(PIECES_START_POSITION);
```
Zeby bylo wiadomo, ze na poczatku chodzi zawsze o pozycje startowa.